# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/src/renderer.rs
# Commit: 5506d1803e15
# Full Hash: 5506d1803e158df5504bc74a7818cb78dca6420c
# Author: Kris Taeleman <ktaeleman@mozilla.com>
# Date: 2019-10-16 05:20:00
# Description:
#   Bug 1585801 - Fixing crash when folder is not writable and refactored logic. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D48999
# ==============================================================================

diff -r f24ddb55b48b -r 5506d1803e15 gfx/wr/webrender/src/renderer.rs
--- a/gfx/wr/webrender/src/renderer.rs	Sat Oct 12 23:50:45 2019 +0000
+++ b/gfx/wr/webrender/src/renderer.rs	Tue Oct 15 17:03:22 2019 +0000
@@ -2791,7 +2791,8 @@
 
     fn handle_debug_command(&mut self, command: DebugCommand) {
         match command {
-            DebugCommand::EnableDualSourceBlending(_) => {
+            DebugCommand::EnableDualSourceBlending(_) |
+            DebugCommand::SetTransactionLogging(_) => {
                 panic!("Should be handled by render backend");
             }
             DebugCommand::FetchDocuments |